src/index.ts

```diff
@@ -406,7 +411,9 @@ export async function scrapeOrganisation(config: ReturnType<typeof loadConfig>,
 	return allDeps
 }
 
-export async function getJsonStructure(accessToken: string, config: Configuration, toUse: string[] | null = null){
+export async function getJsonStructure(accessToken: string, config: Configuration, toUse: string[] | null = null, crawlStart: string | null = null){
```

Comment:
What is the purpose of `crawlStart`?
Why is it an argument? 
Why not just directly write to file?

Reply:
It allows the start time to be specified rather than being determined by the function, which means the exact start time (as recorded in the file) can be known before the crawling completes.